members/WIL000380.yaml

```diff
@@ -10,20 +10,12 @@ contact_form:
           value: $NAME_FULL
           required: true
         - name: Address
-          selector: div .address input
-          value: $ADDRESS_STREET
-          required: true
-        - name: City
-          selector: div .city input
-          value: $ADDRESS_CITY
-          required: true
-        - name: Zip
-          selector: div .zip input
-          value: $ADDRESS_ZIP5
+          selector: textarea#ba762976-cdab-4b64-a563-9d5e27f4c0d9
```

Comment:
is there a selector to use that is less generated looking? I worry this will randomly break on us

Reply:
found one 🎉